lib/honeybadger/http_adapter/hackney.ex

```diff
@@ -33,14 +33,23 @@ defmodule Honeybadger.HTTPAdapter.Hackney do
   end
 
   defp decode(headers, body, opts) when is_binary(body) do
-    case List.keyfind(headers, "content-type", 0) do
-      {"content-type", "application/json" <> _rest} ->
+    content_type =
+      headers
+      |> Enum.find_value(fn {k, v} ->
+        if String.downcase(k) == "content-type", do: String.downcase(v)
+      end)
```

Comment:
Curious; why replace `keyfind` here?

Reply:
That makes sense then. Req will downcase and normalize all headers, and I _thought_ hackney did as well, but apparently not?